packages/truffle/test/scenarios/db/integration.js

```diff
@@ -0,0 +1,177 @@
+const path = require("path");
+const fs = require("fs/promises");
+const Db = require("@truffle/db");
+const { assert } = require("chai");
+const CommandRunner = require("../commandRunner");
+const Ganache = require("ganache");
+const sandbox = require("../sandbox");
+const gql = require("graphql-tag");
+const { Resolver } = require("@truffle/resolver");
+let config, server1, server2, run;
+
+describe("truffle db", function () {
+  // we really don't need to test this when running CI against Geth - there is
+  // also a more complex setup since it requires 2 test nets to be running
+  if (process.env.GETH) return;
```

Comment:
Should we eventually test against GETH? Though, not in this PR.

Reply:
No. The purpose of this test is not to test migrations against Geth but rather to test that db is recording the proper data and can fetch it.